src/styles/_catalog.scss

```diff
@@ -1,9 +1,8 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
-  margin: 0 auto;
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(4, 200px);
+  gap: 48px;
 
```

Comment:
There are should be styles for screen < 488px, you chose mobile first so styles should look like that:

````scss
.catalog {
  // styles for the screen from 0 to first media (488px)
  
  @media (min-width: 488px) {
    //styles for screen from 488px to next media (768)
  }
  
  @media (min-width: 768px) {
     //styles for screen from 768px to next media
  }
}

````